bin/openssh-ldap-publickey

```diff
@@ -212,7 +217,8 @@ unless (
 my $log_level = $conf->{'openssh_ldap_loglevel'};
 my $logfile = $conf->{'openssh_ldap_logfile'} || $fallback_logfile;
 my $timeout = $conf->{'timeout'} || 10;
-my $nss_base_passwd = $conf->{'nss_base_passwd'} || [ { nss_base_passwd => "ou=People," .$conf->{'base'}, scope => undef } ];
+my $nss_base = $conf->{'nss_base'} || "";
```

Comment:
I think the new default value should be equal to the old one, otherwise some script consumers found itself in position when LDAP authorization is suddenly stopped working after the update.